docker-compose.yml

```diff
@@ -38,6 +38,14 @@ services:
     environment:
       SPACETIMEDB_FLAMEGRAPH_PATH: ../../../../flamegraphs/flamegraph.folded
       SPACETIMEDB_TRACY: 1
+      # Tracy utilizes the Time Stamp Counter (TSC) in CPUs for high-resolution timing.
+      # The invariant TSC is a feature that ensures the TSC runs at a constant rate,
+      # regardless of power-saving modes or frequency changes.
+      #
+      # This flag will instruct Tracy to proceed without verifying the presence of an invariant TSC.
+      # If the CPU lacks this feature, Tracy cannot guarantee accurate timing.
+      # So use this approach with caution, as it may affect profiling accuracy.
+      TRACY_NO_INVARIANT_CHECK: 1
```

Comment:
The comment here is super helpful but let's have this option commented out by default